core/templates/pages/exploration-editor-page/ckeditor_dependency.html

```diff
@@ -0,0 +1,25 @@
+<!--Preload the styles required for ckeditor.-->
+<link rel="preload" href="/third_party/static/ckeditor-bootstrapck-1.0.0/skins/bootstrapck/editor.css?t=J5S9" as="style">
+<link rel="preload" href="/third_party/static/ckeditor-4.12.1/plugins/scayt/dialogs/dialog.css?t=J5S9" as="style">
+<link rel="preload" href="/third_party/static/ckeditor-4.12.1/plugins/tableselection/styles/tableselection.css" as="style">
+
+<!--Preload the ckeditor rte-components thumbnail.-->
+<link rel="preload" type="image/png" href="/extensions/ckeditor_plugins/pre/icons/pre.png?t=J5S9" as="image">
+<link rel="preload" type="image/png" href="/third_party/static/ckeditor-bootstrapck-1.0.0/skins/bootstrapck/icons_hidpi.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Collapsible/Collapsible.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Image/Image.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Link/Link.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Tabs/Tabs.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Video/Video.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Collapsible/Collapsible.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Math/Math.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Skillreview/Skillreview.png" as="image">
+<link rel="preload" type="image/png" href="/extensions/rich_text_components/Svgdiagram/Svgdiagram.png" as="image">
```

Comment:
@vojtechjelinek Can you PTAL at this.
Here I am preloading the thumbnails of the rte-components in CKEditor so that when we open the CKEditor while being offline, these images can be rendered. The problem with this is that these images are in png format only and as I am preloading these in the exploration-editor-mainpage.html, [this lighthouse test is failing ](https://github.com/oppia/oppia/pull/13407/checks?check_run_id=3089509442) showing this error:
![image](https://user-images.githubusercontent.com/56637531/126050209-b07bfa68-7c30-4b7a-a336-ce00f61d5cf3.png)
I am not sure what count 3 indicates here as the images I am preloading are more than 3.

I tried solving this error by using the webp image format but for that the only possible way is to check using the javascript whether the browser supports webp, if yes then preload webp otherwise preload png format images. It would be similar to [this.](https://stackoverflow.com/questions/66857139/webp-jpg-preloading-only-the-needed-type-in-a-html-file). But this method does not seem ideal. 
So is it fine to skip the check for these specific images? And also these images are loaded and used in png format only at present.

/cc @kevintab95

Reply:
Replied to this through email.